packages/switch/src/mwc-switch-base.ts

```diff
@@ -63,29 +70,95 @@ export class SwitchBase extends FormElement {
     };
   }
 
-  get ripple() {
-    return this.rippleNode.ripple;
+  protected rippleHandlers: RippleHandlers = new RippleHandlers(() => {
+    this.shouldRenderRipple = true;
+    return this.ripple;
+  });
+
+  protected renderRipple() {
```

Comment:
Currently this if statement is unsupported in our internal SSR tooling. Can you please refactor this fn to be the same as in checkbox as an ugly ternary? Eventually we should support this pattern, but the tooling is under development.